django/santropolFeast/santropolFeast/settings.py

```diff
@@ -160,3 +160,5 @@
     BASE_DIR + '/santropolFeast/static/',
 )
 STATIC_URL = '/static/'
+MEDIA_ROOT = BASE_DIR + '/santropolFeast/static/media/'
```

Comment:
Be carefull: If you set media in static you have a double import.

Static: http://localhost:8000/static/media/avatars/caroline_4nJQLMO.jpeg
Media: http://localhost:8000/media/avatars/caroline_4nJQLMO.jpeg